packages/@lwc/integration-karma/test/light-dom/scoped-slot/index.spec.js

```diff
@@ -61,7 +61,7 @@ describe('scoped slots', () => {
         // For standard slot content, "slot" attribute goes directly on the element unlike scoped
         //  slots where the attribute goes on the template tag
         expect(child.querySelector('.slotname3').innerHTML).toBe(
-            `${vFragBookend}<p slot="slotname3">MLB</p>${vFragBookend}`
+            `${vFragBookend}<p>MLB</p>${vFragBookend}`
```

Comment:
Since the light DOM slot mapping changes based on the `slot` element, the `slot` attribute was removed from this element. 

It still renders in the same position though.